java/client/src/main/java/glide/api/models/commands/GetExOptions.java

```diff
@@ -0,0 +1,111 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import static glide.api.models.commands.GetExOptions.ExpiryType.MILLISECONDS;
+import static glide.api.models.commands.GetExOptions.ExpiryType.PERSIST;
+import static glide.api.models.commands.GetExOptions.ExpiryType.SECONDS;
+import static glide.api.models.commands.GetExOptions.ExpiryType.UNIX_MILLISECONDS;
+import static glide.api.models.commands.GetExOptions.ExpiryType.UNIX_SECONDS;
+
+import glide.api.commands.StringBaseCommands;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Optional arguments to {@link StringBaseCommands#getex(String, GetExOptions)} command.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/getex/">redis.io</a>
+ */
+public class GetExOptions {
+
+    /** Expiry type for the time to live */
+    private final ExpiryType type;
+
+    /** The amount of time to live before the key expires. */
+    private Long count;
+
+    private GetExOptions(ExpiryType type) {
+        this.type = type;
+    }
+
+    private GetExOptions(ExpiryType type, Long count) {
+        this.type = type;
+        this.count = count;
+    }
+
+    /**
+     * Set the specified expire time, in seconds. Equivalent to <code>EX</code> in the Redis API.
+     *
+     * @param seconds time to expire, in seconds
+     * @return Expiry
```

Comment:
Same suggestion elsewhere in this doc
```suggestion
     * @return The options specifying the given expiry.
```